ng-dev/pr/merge/merge-pull-request.ts

```diff
@@ -51,7 +51,7 @@ export async function mergePullRequest(prNumber: number, flags: PullRequestMerge
 
   /** Performs the merge and returns whether it was successful or not. */
   async function performMerge(
-    validationConfig: PullRequestValidationConfig = PullRequestValidationConfig.create({
+    validationConfig = createPullRequestValidationConfig({
```

Comment:
We'd need to read the pull request configuration to get the override values here.